packages/flame/test/components/text_box_component_test.dart

```diff
@@ -37,5 +37,39 @@ void main() {
           ),
       );
     });
+
+    flameGame.test(
+      'internal image is disposed when component is removed',
+      (game) async {
```

Comment:
```suggestion
      (game) {
```

Reply:
Haha I shouldn't have committed this right before I went to bed, it's needed due to the `await`.